x/scheduler/keeper/exported/exported.go

```diff
@@ -0,0 +1,20 @@
+package exported
```

Comment:
This file is not required. You can delete this.

Reply:
deleted